monero-outreach-docs/translations/fr/archive/how_to_buy_monero_fr.md

```diff
@@ -0,0 +1,131 @@
+
+Les cartes de crédits prépayées peuvent fournir un anonymat tant qu’il n’y a pas de métadonnées qui vous relient aux dites cartes. Si vous achetez une carte prépayée avec du liquide et ne l’utilisez pour aucun autre achat, vous aurez un niveau raisonnable de confidentialité. 
+
+_**Avantages**_
+- Pratique
+
+_**Inconvénients**_
+- Non confidentiel
+- Centralisé
+- Forte chance de violation de vos données
+
+### _Acheter des Monero par transfert bancaire_
+---
+
+Les transferts bancaires sont les plus couramment utilisés pour les transactions OTC (sur le marché hors cote). Il est intéressant d’envisager un virement bancaire car il se distingue d’un achat par carte de crédit/débit et ce même si c’est la même banque qui propose les trois services. Ceci car le routage des données est différent. Les transactions par carte de Crédit/débit passe par les mêmes processeurs de paiement et réseaux que n’importe quelle autre carte de crédit. Ceci est similaire au fait que les services permettant le transfert d’emails tendent à être exécutés par des réseaux de distribution tierces. Mais un transfert de compte bancaire à un autre ou plus généralement les transferts par virement bancaire suivent un autre chemin. Un transfert sur des comptes propres à une banque est effectué au sein du réseau de la banque, ce qui signifie que les données de la transaction associées existent en moins d’endroits et la transaction sera finalisée instantanément. Cependant, un virement bancaire introduit de nouveaux groupes de surveillances, particulièrement dans la banque receveuse et les banques intermédiaires, mais aussi SWIFT (Société pour la télécommunication financière inter-banque mondiale). Si votre virement a pour destination ou est émis des États Unis, il passera aussi par le bureau de contrôle des actifs étrangers et par FedWire.
+
+_**Avantages**_
+- Adéquats pour de gros montants
+
+_**Inconvénients**_
+- Non confidentiel
+- Centralisé
+- Cher et peu pratique
+
+### _Acheter des Monero via PayPal_
+---
+Les services du type PayPal, Apple Pay, Google Pay, Venmo ou Western Union ne sont pas confidentielles et peuvent être couteux. Ces services de transferts d’argent auront aussi leurs propres conditions d’utilisation et règles, pouvant vous exposer à des risques additionnels. Avec PayPal en particulier, les vendeurs de XMR doivent être prudent car les transactions peuvent être annulées et PayPal a pour règle de s’aligner sur l’acheteur en cas de litige. Dans l’exemple de scam suivant, Alice s’accorde pour acheter 1 XMR pour un montant de X Dollars à Bob. Alice envoie à Bob les X dollars via PayPal, et ensuite Bob envoie à Alice 1 XMR. Alice produit un avis de litige avec PayPal et les X dollars se retrouve remboursés, alors qu’elle garde les Monero immuables.
+
+_**Avantages**_
+- Pratique
+
+_**Inconvénients**_
+- Pas confidentielle
```

Comment:
confidentielle ==> **confidentiel**; les autres adjectifs ci-bas sont masculins, gardons tout au masculin

Reply:
Ok. I took the opportunity to also change all of the 'Non confidentiel' (which appeared in a few occasions and listed as 'Inconvénients') into 'Pas confidentiel'.